### PR TITLE
Firebase removed for client-side telemetry & segment implemented for client side telemetry

### PR DESCRIPTION
**Pull request by PraveenCuebook**

### CODE REVIEW

File: api/anomaly/services/telemetry.py [hunk 1]
```diff
@@ -94,3 +104,36 @@ def rca_event_log(status):
         })
 
 
+def getInstallationId():
+    """ Function to get installation, if exists or create one"""
+    try:
+        res = ApiResponse()
+        from anomaly.models import InstallationTable
```
Comment (vincue) [inline api/anomaly/services/telemetry.py:111]: import at top

File: ui/src/services/installation.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import apiService from "./api";
+import { message , notification} from "antd"
+
+class Installation {
+    async getInstallationId(){
+        const response = await apiService.get("anomaly/installationId")
+        return response
```
Comment (vincue) [inline ui/src/services/installation.js:7]: use `.then` & `.catch`

Comment (PraveenCuebook) [reply ui/src/services/installation.js]: done

File: api/anomaly/services/telemetry.py (hunk 1 above)
Comment (PraveenCuebook) [reply api/anomaly/services/telemetry.py]: done

**Review by vincue — APPROVED** [review]

**Pull request merged**